front-end/src/ui-components/GeneralChainSignalBar.tsx

```diff
@@ -0,0 +1,67 @@
+// Copyright 2019-2020 @paritytech/polkassembly authors & contributors
+// This software may be modified and distributed under the terms
+// of the Apache-2.0 license. See the LICENSE file for details.
+
+import styled from '@xstyled/styled-components';
+import React from 'react';
+import { Progress } from 'semantic-ui-react';
+
+interface Props {
+	className?: string
+	ayeSignals: number
+	naySignals: number
+}
+
+const GeneralChainSignalBar = ({ ayeSignals, className, naySignals }: Props) => {
+	let percent = 0;
+	const total = ayeSignals + naySignals;
+	if (total > 0) {
+		percent = ayeSignals/total*100;
+	}
+
+	return (
+		<div className={className}>
+			<div className='ayes'>{ayeSignals === 1 ? '1 Aye' : `${ayeSignals} Ayes`}</div>
+			<div className='nays'>{naySignals === 1 ? '1 Nay' : `${naySignals} Nays`}</div>
+			<Progress
```

Comment:
Note to anyone willing to do it (@erler ?) I think we should show the percentage of "Aye" to make it easy to read.

Reply:
Added in parentheses, e.g. "1 Aye (25%)"